src/components/navBar/navBar.js

```diff
@@ -211,22 +211,25 @@ MdNavBarController.prototype._updateTabs = function(newValue, oldValue) {
   var tabs = this._getTabs();
   var oldIndex = -1;
   var newIndex = -1;
-  var newTab = this._getTabByName(newValue);
-  var oldTab = this._getTabByName(oldValue);
 
-  if (oldTab) {
-    oldTab.setSelected(false);
-    oldIndex = tabs.indexOf(oldTab);
-  }
+  if (tabs) {
```

Comment:
To prevent nesting, we can change this to `if (!tabs) return;`. We can also move it up to before `oldIndex` and `newIndex` are initialized.